cdap-docs/admin-manual/source/installation/cloudera/installation.rst

```diff
@@ -61,7 +61,7 @@ Run the Cloudera Manager Admin Console *Add Service* Wizard and select *CDAP*.
 When completing the Wizard, these notes may help:
 
 - *Add Service* Wizard, Page 2: **Optional Hive dependency** is for the optional CDAP
```

Comment:
If explore is enabled by default, Hive is needed by default. Saying "Optional Hive dependency" makes this unclear.